src/modules/dailyCreditsSummary/i18n/sv_SE.ts

```diff
@@ -0,0 +1,106 @@
+            backgroundColor: '#ff2800',
+            textColor: 'white',
+        },
+        mission: {
+            regex: '^%missions%( \\(Fire Alarm System\\))?( - False Alarm)?$',
+            title: 'Egna uppdrag',
+            backgroundColor: '#ff2400',
+            textColor: 'white',
+        },
+        cancelledMissions: {
+            regex: '^%missions%( \\(Fire Alarm System\\))? - [cC]anceled$',
+            title: 'Avbrutna uppdrag',
+            backgroundColor: '#7C7978',
+            textColor: 'white',
+        },
+        buyVehicle: {
+            regex: /Vehicle bought/,
+            backgroundColor: '#007fff',
+            textColor: 'white',
+        },
+        dailyLogin: {
+            regex: /Daily login reward/,
+            backgroundColor: '#8db600',
+            textColor: 'black',
+        },
+        buildings: {
+            regex: /(Building constructed|Refund Building$|Building demolished without refund)/,
+            title: 'Byggda/rivna byggnader',
+            backgroundColor: '#ed872d',
+            textColor: 'black',
+        },
+        upgradeBuilding: {
+            regex: /(Station (upgraded( \(from small .*? station\))??|constructed)|Extended guard|Cancel:|Refund Building upgrade)/,
+            title: 'Uppgraderade stationer',
+            backgroundColor: '#000080',
+            textColor: 'white',
+        },
+        task: {
+            regex: /Completed task («.*?»|".*?")/,
+            title: 'Avslutad uppgift',
+            backgroundColor: '#bb3385',
+            textColor: 'white',
+        },
+        patients: {
+            regex: /(Patient Treatment|Patient Transport)/,
+            title: 'Patient',
+            backgroundColor: '#fff600',
+            textColor: 'black',
+        },
+        alliancePatients: {
+            regex: /Hospital - Alliance/,
+            backgroundColor: '#ffff99',
+            textColor: 'black',
+        },
+        alliancePrisoners: {
+            regex: /Police - Alliance/,
+            backgroundColor: '#ffff99',
+            textColor: 'black',
+        },
+        prisoners: {
+            regex: /Prisoner? Transported/,
+            title: 'Fånge',
+            backgroundColor: '#0bda51',
+            textColor: 'black',
+        },
+        schoolings: {
+            regex: /(education|Alliance Training Applicant)/,
+            title: 'Utbildning',
+            backgroundColor: '#ff00ff',
+            textColor: 'white',
+        },
+        shitComplexes: {
+            regex: /(Attached building to Building Complex Base|Upgraded to Building Complex Base|Upgraded to Building Complex|Building Complex Base upgrade finished)/,
+            title: 'Byggnadskomplex',
+            backgroundColor: '#b5651d',
+            textColor: 'white',
+        },
+        fireAlarmSystemCancel: {
+            regex: /False Alarm/,
+            titel: 'Falska larm',
+            backgroundColor: '#c80815',
+            textColor: 'white',
+        },
+    },
```

Comment:
This is fine for first MR but will need to be adjusted later. Otherwise the filters will not work ;)

Reply:
can u please explain what needs to adjusted?

Reply:
or alteast the basics of it so i can go thru it again

Reply:
all of the `regex`es need to be adjusted so that they match the entries of https://www.larmcentralen-spelet.se/credits/daily

Reply:
Oh you mean like the non lss site?
yeah that i can take a look at tomorrow:)